defaults/main.yml

```diff
@@ -43,12 +42,12 @@ kpi_git_key_ssh_file:
 kpi_git_key_filename: id_ed25519
 kpi_copy_key_from_file: false
 kpi_remove_git_key: true
-kpi_python_source_version: 2.7
-kpi_python_version: "python2.7"
+kpi_python_source_version: "3"
+kpi_python_version: python{{ kpi_python_source_version }}
 kpi_checkout_path: "{{ kpi_versioned_path }}/{{ kpi_git_branch }}"
 kpi_app_path: "{{ kpi_system_user_home }}/app"
 kpi_versioned_path: "{{ kpi_app_path }}-versioned"
-kpi_git_branch: ona-custom-changes
+kpi_git_branch: kpi_2.0
```

Comment:
Should we bake this into the default variables?

Reply:
Hey @moshthepitt 
No this will not be used in the role's default vars. I was using this with testing the config as this branch has been rebased with the default repo.
Also I may need to deploy this updated branch(alongside the `ona-custom-changes` branch), so to build this image, this is the branch I want to be pointing to for this test.

However, when the branch is fully tested, it will be merged back into the `ona-custom-changes` branch, so the playbooks inventories will still point to this branch